tests/test_Consumer.py

```diff
@@ -14,6 +14,8 @@ def test_basic_api():
         kc = Consumer()
     except TypeError as e:
         assert str(e) == "expected configuration dict"
+    else:
```

Comment:
should use http://doc.pytest.org/en/latest/reference.html#pytest-raises

Reply:
Yeap that will be better.